### ISSUE TITLE
Moving repos from gh/lxde to gh/lxqt

### ISSUE THREAD

**Issue by agaida** [lxqt/lxqt#1464]
##### Context

I will move the LXQt repos today to their new home gh/lxqt - github will redirect to the new location for a while, the fastest way to fix local clones is
```
$EDITOR .git/config
[remote "origin"]
        url = git@github.com:lxde/$foo
-->     url = git@github.com:lxqt/$foo
```
or 
```
git remote set-url origin git@github.com:lxqt/$foo
```

**Comment by agaida** [comment]
All repos are moved - right now there is only one group called LXQt with no repos in it - all LXQt members have standard RW-access to the repos. if we need more fine grained rules please ping me.

**Comment by yan12125** [comment]
Great job! Here's a breakage: some LXDE repos (libfm, lxmenu-data, menu-cache) are referenced in https://github.com/lxqt/lxqt/blob/master/.gitmodules, so ```git clone --recursive https://github.com/lxqt/lxqt``` doesn't work.

**Comment by agaida** [comment]
and we messed up our website(s) - thanks for mention it, normal fallout i would guess. Will take care of.

**Comment by agaida** [comment]
should be fixed with lxqt/lxqt/pull/1465
